skyportal/handlers/api/group.py

```diff
@@ -27,15 +27,9 @@ def has_admin_access_for_group(user, group_id):
         .filter(GroupUser.user_id == user.id)
         .first()
     )
-    return (
-        len(
-            {"System admin", "Manage users", "Manage groups"}.intersection(
-                set(user.permissions)
-            )
-        )
-        > 0
-        or groupuser.admin
-    )
+    return {"System admin", "Manage users", "Manage groups"}.intersection(
```

Comment:
Did I lead you "from the road into the ditch" as they say in Afrikaans?  I guess this now needs a bool() to always evalute to True/False :grimacing: